### DISCUSSION TITLE
Upgrade to Angular 13 planned?

### DISCUSSION THREAD

**Discussion by Thomas-Schindler** [getsaf/shallow-render#211] [Q&A] [answered]
Hello,

We are using shallow-render with jest as our tool for unit test in our Angular projects. We are about to switch to Angular 13 now, but discovered, that shallow-rende has a peer-dependency to @angular/common@"12.x". Which gives us the following error when we run "npm install" in our workspace with Angular 13 and shallow-render 12.0.1:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: empic-med-nx@0.0.0
npm ERR! Found: @angular/common@13.0.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^13.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"12.x" from shallow-render@12.0.1
npm ERR! node_modules/shallow-render
npm ERR!   dev shallow-render@"^12.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Are there any plans to migrate the peer dependencies in the near future?

Kindes Regards
Thomas

**Comment by getsaf** [comment, marked as answer]
For sure. This usually only takes a half hour or so I just haven't been keeping track of Angular releases. I'll get this done over the weekend unless you have time to try and submit the pr. It's usually just a matter of bumping the Angular dependencies in the package.json and running the build:all step to see that it passes.
